libhoth_dbus.c

```diff
@@ -0,0 +1,253 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "libhoth_dbus.h"
+
+#include <stdlib.h>
+#include <systemd/sd-bus.h>
+
+#include "libhoth.h"
+
+#define SERVICE "xyz.openbmc_project.Control.Hoth"
```

Comment:
nit: Rename these macros to something more unique to avoid future potential conflict?